src/components/manager/access/Subject.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+
+import { Popover, Typography } from "antd";
+
+import { stringifyJSONRenderIfMultiKey } from "./utils";
+
+const Subject = ({ subject, groupsByID }) => {
+    const currentIDToken = useSelector((state) => state.auth.idTokenContents);
+
+    const { sub, client, iss, group, everyone } = subject;
+
+    if (sub || client) {
+        return (
+            <p style={{ margin: 0, lineHeight: "1.6em" }}>
+                <strong>{sub ? "Subject" : "Client"}:</strong>{" "}
+                <Typography.Text code={true}>{sub ?? client}</Typography.Text><br />
+                <strong>Issuer:</strong>{" "}
+                <Typography.Text code={true}>{iss}</Typography.Text><br />
+                {(sub === currentIDToken?.sub && iss === currentIDToken?.iss) ? <><em>(this is you)</em></> : null}
+            </p>
+        );
+    } else if (group) {
```

Comment:
In which case would a subject have no `sub` or `client`, but would have a `group`?

Reply:
there are four subject possibilities from the bento authorization service:

```
{iss, sub}
{iss, client}
{group: #}
{everyone: true}
```

in this case, the group definition itself contains a list of iss+(sub/client) or an expression which is evaluated on the token